posthog/urls.py

```diff
@@ -404,9 +403,9 @@ def delete_events(request):
     urlpatterns.append(path("delete_events/", delete_events))
 
 # Routes added individually to remove login requirement
-frontend_unauthenticated_routes = ["preflight", "signup"]
+frontend_unauthenticated_routes = ["preflight", "signup", r"signup\/[A-Za-z0-9\-]*"]
 for route in frontend_unauthenticated_routes:
-    urlpatterns.append(path(route, home))
+    urlpatterns.append(re_path(route, home))
```

Comment:
Would this work with `opt_slash_path`? Would be best if it did, as currently just appending a slash results in a 404.

Reply:
I think this is fine because the front-end router doesn't support the optional trailing slash either (on these routes or pretty much any other route), so you still get a 404.

Reply:
Ah, that's true :(